diff -Naur tenshi-0.14.orig/Makefile tenshi-0.14/Makefile
--- tenshi-0.14.orig/Makefile	2014-03-24 16:32:09.937824990 +0400
+++ tenshi-0.14/Makefile	2014-03-24 16:33:28.880829538 +0400
@@ -5,12 +5,12 @@
 
 bindir = /usr/sbin
 sysconfdir = /etc
-docdir = /usr/share/doc/tenshi-${VERSION}
+docdir = /usr/share/doc/tenshi
 mandir = /usr/share/man
 libdir = /var/lib/tenshi
 
 DOCS = README INSTALL CREDITS LICENSE Changelog FAQ
-SAMPLES = tenshi.conf tenshi.debian-init tenshi.gentoo-init tenshi.solaris-init tenshi.redhat-init tenshi.suse-init tenshi.redhat-spec tenshi.suse-spec
+SAMPLES = tenshi.conf tenshi.redhat-init
 BIN = Makefile tenshi
 MAN = tenshi.8
 
@@ -35,10 +35,10 @@
 install:
 	install -D tenshi ${DESTDIR}${bindir}/tenshi
 	[ -f ${DESTDIR}${sysconfdir}/tenshi/tenshi.conf ] || \
-		install -g root -m 0644 -D tenshi.conf ${DESTDIR}${sysconfdir}/tenshi/tenshi.conf
-	install -d ${DESTDIR}${docdir}
-	install -m 0644 ${DOCS} ${DESTDIR}${docdir}/
+		install -m 0644 -D tenshi.conf ${DESTDIR}${sysconfdir}/tenshi/tenshi.conf
+#	install -d ${DESTDIR}${docdir}
+#	install -m 0644 ${DOCS} ${DESTDIR}${docdir}/
 	[ -d ${DESTDIR}${mandir}/man8 ] || \
 		install -d ${DESTDIR}${mandir}/man8
-	install -g root -m 0644 tenshi.8 ${DESTDIR}${mandir}/man8/
-	install -g root -m 755 -d ${DESTDIR}${libdir}
+	install -m 0644 tenshi.8 ${DESTDIR}${mandir}/man8/
+	install -m 755 -d ${DESTDIR}${libdir}
